src/providers/registry.ts

```
@@ -6,12 +6,13 @@
~import { MemoryPoisoningProvider } from '../redteam/providers/agentic/memoryPoisoning';
~import RedteamBestOfNProvider from '../redteam/providers/bestOfN';
~import RedteamCrescendoProvider from '../redteam/providers/crescendo';
~import RedteamCustomProvider from '../redteam/providers/custom';
~import RedteamGoatProvider from '../redteam/providers/goat';
~import RedteamIterativeProvider from '../redteam/providers/iterative';
 import RedteamImageIterativeProvider from '../redteam/providers/iterativeImage';
 import RedteamIterativeTreeProvider from '../redteam/providers/iterativeTree';
 import RedteamPandamoniumProvider from '../redteam/providers/pandamonium';
+import RedteamCustomProvider from '../redteam/providers/playbook';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'RedteamCustomProvider'. Consider to delete it or rename it.

'RedteamCustomProvider' is defined here:


(lint/suspicious/noRedeclare)